packages/safe-apps-sdk/jest/setup.ts

```diff
@@ -0,0 +1,24 @@
+import crypto from 'crypto';
+
+function getRandomValues(buf: Uint8Array) {
+  if (!(buf instanceof Uint8Array)) {
+    throw new TypeError('expected Uint8Array');
+  }
+  if (buf.length > 65536) {
+    const e = new Error();
+    e.message =
+      "Failed to execute 'getRandomValues' on 'Crypto': The " +
+      "ArrayBufferView's byte length (" +
+      buf.length +
+      ') exceeds the ' +
+      'number of bytes of entropy available via this API (65536).';
+    e.name = 'QuotaExceededError';
+    throw e;
+  }
+  const bytes = crypto.randomBytes(buf.length);
```

Comment:
Why not use Math.random?

Reply:
if it is because Math.random is not random enought you can just hash `new Date().getTime()` with it.

So something like `sha256((new Date()).getTime().toString() + randomString)`

random string could be based on https://stackoverflow.com/questions/1349404/generate-random-string-characters-in-javascript

In general I don't mind ... but I had issue with crypto before during development because it was not available in "unsecure" environments. (https://dev.to/hoffmann/cannot-read-property-digest-of-undefined-51np)

Reply:
Ohhh actually the `getRandomValues` is the only method always available ... so we can ignore that.

https://developer.mozilla.org/en-US/docs/Web/API/Window/crypto
```
Although window.crypto is available on all windows, the returned Crypto object only has one usable feature in insecure contexts: the getRandomValues() method. In general, you should use this API only in secure contexts.
```

Reply:
> So something like sha256((new Date()).getTime().toString() + randomString)
>
> random string could be based on https://stackoverflow.com/questions/1349404/generate-random-string-characters-in-javascript

It's more complex than the approach I took, isn't it? So currently there's only `randomString` part which is generated via `getRandomValues`